packages/devtools_app/lib/src/screens/vm_developer/object_inspector/vm_code_display.dart

```diff
@@ -318,7 +318,7 @@ class VmCodeDisplay extends StatelessWidget {
   }
 
   /// Returns a list of key-value pairs (map entries)
-  /// containing detailed information of a VM Func object [function].
+  /// containing detailed information of a VM Func object `function`.
```

Comment:
out of scope of this PR, but this dart doc looks incorrect. Maybe a copy paste error at some point.

Reply:
I can delete the text or change it, since we're looking at it. Or I can land as is.

Reply:
If you don't mind, since you're already here? Looks like this should be referencing CodeObject instead